x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/github_app.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiPanel, EuiSpacer } from '@elastic/eui';
+
+import { AppLogic } from '../../../../app_logic';
+import {
+  WorkplaceSearchPageTemplate,
+  PersonalDashboardLayout,
+} from '../../../../components/layout';
+import { NAV, SOURCE_NAMES } from '../../../../constants';
+
+import { staticSourceData } from '../../source_data';
+
+import { AddSourceHeader } from './add_source_header';
+import { SourceFeatures } from './source_features';
+
+export const GitHubApp: React.FC = () => {
+  const { isOrganization } = useValues(AppLogic);
+
+  const name = SOURCE_NAMES.GITHUB;
+  const data = staticSourceData.find((source) => (source.name = name));
+  const Layout = isOrganization ? WorkplaceSearchPageTemplate : PersonalDashboardLayout;
```

Comment:
We probably won't need this, but I don't see the harm in keeping it.

Reply:
Yeah, I thought so too, but wanted to future-proof it for when we roll out all users. Thanks!